src/gd_kotlin.cpp

```diff
@@ -91,10 +91,28 @@ void GDKotlin::set_jvm_options() {
 
 #include <core/io/dir_access.h>
 
+#ifdef __ANDROID__
+#include <unistd.h>
+#endif
+
 String GDKotlin::copy_new_file_to_user_dir(const String& file_name) {
     String file_res_path {String(BUILD_DIRECTORY) + file_name};
     String file_user_path {String(USER_DIRECTORY) + file_name};
 
+#ifdef __ANDROID__
```

Comment:
More code can be shared between the 2 preprocessors blocks. Also we don't always copy those files, only when we detect a change in the file, is that a requirement that Android makes a fresh copy each time now ?

Reply:
As i tried to convey in the comment one line below, android recommends copying dynamically loaded content each time the app starts: https://developer.android.com/about/versions/14/behavior-changes-14#handle-existing-files